### PR TITLE
Updated release notes and holographic remoting docs

### PR DESCRIPTION
**Pull request by RogPodge**
Added back previously deleted section in release notes regarding the workaround for the MsBuild with unity 2019.3+. Also updated holographic remoting document

## Changes
- Mainly comes as a way to mitigate this issue #7830, will leave it open so long as the other related issues are open

### CODE REVIEW

**Comment by RogPodge** [comment]
@davidkline-ms suggested adding this guidance to the root README, and though I agree that we should reduce the pain points for Unity 2019 users, I'm not sure its the most appropriate location.

**Comment by RogPodge** [comment]
Partially helps address #7845

File: Documentation/Tools/HolographicRemoting.md [hunk 1]
```diff
@@ -23,10 +23,27 @@ configure the project using the following steps.
 1. Ensure that **Enable MSBuild for Unity** is selected
 1. Click **Apply**
 
+When using **Unity 2019.3** and later the **Enable MSBuild for Unity** is not available. please follow the below procedures to enable holographic remoting. 
+
+1. Run the MRTK Configurator Utility (**Mixed Reality Toolkit > Utilities > Configure Unity Project**)
+1. Set the target platform ins **File > Build Settings** to **Universal Windows Platform**
```
Comment (keveleigh) [inline Documentation/Tools/HolographicRemoting.md:29]:
```suggestion
1. Set the target platform in **File > Build Settings** to **Universal Windows Platform**
```

File: Documentation/Tools/HolographicRemoting.md [hunk 2]
```diff
@@ -23,10 +23,27 @@ configure the project using the following steps.
...
+1. Click **Apply**
+1. Open **Window > Package Manager**
+    - Ensure that the **Windows XR Plugin** is not installed, as the legacy **Windows Mixed Reality** package will not function alongside it
```
Comment (keveleigh) [inline Documentation/Tools/HolographicRemoting.md:32]: We might want to clarify that this step is only if you want to run on the legacy XR pipeline.

```suggestion
    - Ensure that the **Windows XR Plugin** is not installed if the project isn't using [XR SDK](../GettingStartedWithMRTKAndXRSDK.md), as the legacy **Windows Mixed Reality** package will not function alongside it
```

**Pull request merged**